src/test/java/seedu/jarvis/model/greeting/GreetingTest.java

```diff
@@ -0,0 +1,66 @@
+package seedu.jarvis.model.greeting;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class GreetingTest {
+    @Test
+    void constructor_createsDefaultGreeting() {
+        Greeting greeting = new Greeting();
+        assertTrue(greeting.toString().equals(""));
+    }
+
+    @Test
+    void isEmpty_emptyGreeting_returnsTrue() {
+        Greeting greeting = new Greeting();
+
+        // When greeting is empty string
+        assertTrue(greeting.isEmpty());
+
+        // When greeting is default message
+        greeting.setGreeting("Hi! Please log in to continue");
+        assertTrue(greeting.isEmpty());
+    }
+
+    @Test
+    void getGreeting() {
```

Comment:
Perhaps this test method signature could be setGreeting_validString_success()?

Reply:
👍